tests/pagix.test.js

```diff
@@ -0,0 +1,16 @@
+const fc = require('fast-check');
+const { pagix } = require('../src/index');
+
+
+describe('pagix', () => {
+	it('should correctly calculate next and prev', () => fc.assert(
```

Comment:
Hmmm, I wrote this pretty late last night. @brunobertolini If this kind of change is something you'd accept I can rewrite the tests to be clearer what it's testing rather than this far too dynamic way of testing.

Basically I'll change to a series of tests checking one thing, eg "should return next = current + 1 for < last page", "should return next = false for >= last page" etc